### PR TITLE
Admin PR, this time atomized for real.

### PR DESCRIPTION
**Pull request by DominikPanic**

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1719,14 +1719,13 @@
 
 
 /mob/living/carbon/human/take_over(mob/M)
-	assigned_squad?.clean_marine_from_squad(src)
+	if(assigned_squad)
+		assigned_squad.clean_marine_from_squad(src)
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/human/human.dm]: Why the format change? The prior one worked the same, but with one less check.

Comment (DominikPanic) [reply code/modules/mob/living/carbon/human/human.dm]: Because I don't trust it :cry:

Comment (Rohesie) [reply code/modules/mob/living/carbon/human/human.dm]: Stop being paranoid.
D=<
This should be changing nothing.

Comment (spookydonut) [reply code/modules/mob/living/carbon/human/human.dm]: i think the new code is more readible

Comment (Rohesie) [reply code/modules/mob/living/carbon/human/human.dm]: Reeeeee.
Two against one, sad times. Oh well.
If you prefer it for readability I won't pry further, just don't assume it doesn't do what it's supposed to do.

File: code/modules/admin/verbs/datumvars.dm [hunk 2]
```diff
@@ -677,566 +679,538 @@
...
-				H.dropItemToGround(H.head)
-				H.head = new /obj/item/clothing/head/kitty(H)
-				H.regenerate_icons()
-				H.head.update_icon(H)
-				log_admin("[key_name(usr)] has purrbated [key_name(H)].")
-				message_admins("[ADMIN_TPMONTY(usr)] has purrbated [ADMIN_TPMONTY(H)].")
+		if(!new_language)
+			return
 
+		if(!L)
```
Comment (Rohesie) [inline code/modules/admin/verbs/datumvars.dm]:
```suggestion
		if(QDELETED(L))
```

**Pull request merged**